internal/mock/cloudsql.go

```diff
@@ -75,6 +75,7 @@ func NewFakeCSQLInstance(project, region, name string, opts ...FakeCSQLInstanceO
 		project:   project,
 		region:    region,
 		name:      name,
+		ipAddrs:   map[string]string{"PUBLIC": "0.0.0.0"},
```

Comment:
nit: Should we define "PUBLIC" and "PRIVATE" in a constant somewhere?

Reply:
I went back and forth on using the same constants from the `cloudsql` package. Since it's constrained to this file and not something test callers will use, it's not terrible to use a string.

Reply:
This will come up again in my next PR. Let's talk about it again to see if it's working as a string.